src/backend/api/Fusion.Resources.Domain/Models/SecondOpinions/QuerySecondOpinion.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Fusion.Resources.Database.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Fusion.Resources.Domain
+{
+    public class QuerySecondOpinion
+    {
+        public QuerySecondOpinion(DbSecondOpinionPrompt secondOpinion)
+        {
+            Id = secondOpinion.Id;
+            Number = secondOpinion.Number;
+            Description = secondOpinion.Description;
+            Title = secondOpinion.Title;
+            CreatedById = secondOpinion.CreatedById;
+            CreatedBy = new QueryPerson(secondOpinion.CreatedBy);
+            CreatedDate = secondOpinion.CreatedDate;
+
+            if(secondOpinion.Responses is not null)
+            {
+                Responses = secondOpinion.Responses.Select(x => new QuerySecondOpinionResponse(x));
+            }
+
+            if(secondOpinion.Request is not null)
```

Comment:
According to the db-model the expression here is always true, so do you need it?